File: 0485-max-consecutive-ones/0485-max-consecutive-ones.cs
public class Solution {
    public int FindMaxConsecutiveOnes(int[] nums) {
       int ans = 0, res = 0;
        foreach (int x in nums) {
            ans = (x == 0) ? 0 : ans + 1;
            res = Math.Max(res, ans);
        }
        return res;
        
    }
}

File: 0485-max-consecutive-ones/0485-max-consecutive-ones.cpp
class Solution {
public:
    int findMaxConsecutiveOnes(vector<int>& nums) {
        int ans = 0; int res = 0;
        for(int x : nums) {
            x == 0 ? ans = 0 : ++ans;
            res = max(res, ans);
        }
        return res;
    }
};